doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1595,11 +1599,11 @@ A subscription is represented by a JSON object with the following fields:
 
 A `subject` contains the following subfields:
 
-+ `entities`: A list of objects, each one composed of the following subfields (`id`/`idPattern` or
-  `type` must be present):
-    + `id` or `idPattern`: Id or pattern of the affected entities. Both cannot be used at the same
-      time, but at least one of them must be present.
-    + `type`: Type of the affected entities (optional).
++ `entities`: A list of objects, each one composed of the following subfields:
+  + `id` or `idPattern`: Id or pattern of the affected entities. Both cannot be used at the same
+     time, but at least one of them must be present.
```

Comment:
at least one of them ...
at least one, but maximum one?
Doesn't make much sense:

    Both cannot be used at the same time, but one of them must be present.

or, more concise:

    Mandatory and mutually exclusive